### PR TITLE
[Hyperdrive] add Supabase example for Hyperdrive

### PR DESCRIPTION
**Pull request by inian**

### CODE REVIEW

File: content/hyperdrive/examples/supabase.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## 1. Allow Hyperdrive access
+
+You can connect Hyperdrive to any existing Supabase database as the Postgres user which is set up during project creation.
+Alternatively, to create a new user for Hyperdrive, run these commands in the [SQL Editor](https://supabase.com/dashboard/project/_/sql/new).
+
+```sql
+CREATE ROLE hyperdrive_user LOGIN PASSWORD 'sufficientlyRandomPassword';
+
+-- Here we are granting it the postgres role. In practice, you want to create a role with lesser privileges.
```
Comment (Maddy-Cloudflare) [inline content/hyperdrive/examples/supabase.md]:
```suggestion
-- Here, you are granting it the postgres role. In practice, you want to create a role with lesser privileges.
```

**Review by Maddy-Cloudflare — APPROVED** [review]

**Review by Maddy-Cloudflare — CHANGES_REQUESTED** [review]
To be addressed.

**Comment by inian** [comment]
@Maddy-Cloudflare I have addressed your comments. can you PTAL?

**Review by elithrar — APPROVED** [review]

**Review by Maddy-Cloudflare — APPROVED** [review]

File: content/hyperdrive/get-started.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ To continue, you will need:
 1. A [Cloudflare account](https://dash.cloudflare.com/sign-up/workers-and-pages) if you do not have one already.
 2. [`npm`](https://docs.npmjs.com/getting-started) installed on your local machine.
 3. [`Node.js`](https://nodejs.org/en/) installed. Use a Node version manager like [Volta](https://volta.sh/) or [nvm](https://github.com/nvm-sh/nvm) to avoid permission issues and change Node.js versions. [Wrangler](/workers/wrangler/install-and-update/) requires a Node version of `16.17.0` or later.
-4. **A publicly accessible PostgreSQL (or PostgreSQL compatible) database**. Cloudflare recommends [Neon](https://neon.tech/) or [Supabase](https://supabase.com) if you do not have an existing database. Read the [Neon's documentation to create your first database](https://neon.tech/docs/introduction) or the [Supabase guide to get started](https://supabase.com/docs/guides/getting-started).
```
Comment (ant-supabase) [inline content/hyperdrive/get-started.md]: Shame! Anything we're missing here to get the rec @elithrar?

Comment (elithrar) [reply content/hyperdrive/get-started.md]: We'd prefer to keep the tutorial PR discrete for now, and avoid making users (during our own getting started guide!) have to figure out what to choose from.

Comment (awalias) [reply content/hyperdrive/get-started.md]: oops wrong account...

**Pull request merged**